_elements/buttons.md

```diff
@@ -8,26 +8,55 @@ title: Buttons
   <!-- Add HTML markup for example here -->
 
   <h3>Primary Buttons</h3>
-  <button>Static</button>
-  <button class="usa-button-active">Active</button>
-  <button class="usa-button-hover">Hover</button>
+  <div class="button_wrapper">
+    <button>Sign up</button>
+    <button class="usa-button-active">Sign up</button>
```

Comment:
@msecret I just talked to @angel and agreed to add the button states added back in. 

Here is a picture for reference to the previous state:
<img width="333" alt="screen shot 2015-08-15 at 12 21 48 pm" src="https://cloud.githubusercontent.com/assets/5249443/9310824/25f229e4-44c7-11e5-9fa0-a055896c24c3.png">